src/components/Card.js

```diff
@@ -29,10 +39,30 @@ export default class Card {
     this._cardElement.remove();
   }
 
-  _handleLikeIcon() {
-    this._cardElement
-      .querySelector(".card__like-button")
-      .classList.toggle("card__like-button_active");
+  setLikeState(isLiked) {
+    const likeButton = this._cardElement.querySelector(".card__like-button");
+    if (isLiked) {
+      likeButton.classList.add("card__like-button_active");
+    } else {
+      likeButton.classList.remove("card__like-button_active");
+    }
+  }
+
+  isLiked() {
+    return this._isLiked;
+  }
+
+  updateIsLiked(isLiked) {
+    this._isLiked = isLiked;
+  }
+
+  toggleLikeIcon(isLiked) {
+    const likeButton = this._cardElement.querySelector(".card__like-button");
+    if (isLiked) {
+      likeButton.classList.add("card__like-button_active");
+    } else {
+      likeButton.classList.remove("card__like-button_active");
```

Comment:
`setLikeState` and `toggleLikeIcon` have the same code. Please, delete one of them